### PR TITLE
Login/Register Animations

### PR DESCRIPTION
**Pull request by JCGarzaa**
Summary
- Login/Register animations

Demo

https://user-images.githubusercontent.com/83436163/180577775-01ad8e40-5bd3-462e-b6a8-21b5421918f6.mov

### CODE REVIEW

File: app/src/main/java/com/example/wrk/LoginActivity.java [hunk 1]
```diff
@@ -51,6 +51,17 @@ public void onClick(View v) {
                 loginUser(username, password);
             }
         });
+        etUsername.setTranslationX(300);
```
Comment (katerina520) [inline app/src/main/java/com/example/wrk/LoginActivity.java:54]: the 300, 1000 numbers are reused accross files, so you could put them in a const

File: app/src/main/java/com/example/wrk/fragments/ProfileFragment.java [hunk 2]
```diff
@@ -236,7 +233,7 @@ private double calcPercentage() throws ParseException {
             double difference = currentUser.getInt("workoutsThisMonth") - averageWorkoutsThisMonth;
             percentDifference = (difference / averageWorkoutsThisMonth) * 100;
         }
-        return percentDifference;
+        return Math.round(percentDifference * 10.0) / 10.0; // round to one decimal
```
Comment (katerina520) [inline app/src/main/java/com/example/wrk/fragments/ProfileFragment.java:236]: would be nice to point out in the PR that you are also modifying small thing unrelated to the main PR content

File: app/src/main/java/com/example/wrk/LoginActivity.java (hunk 1 above)
Comment (JCGarzaa) [reply app/src/main/java/com/example/wrk/LoginActivity.java:54]: Good idea. Will fix this

File: app/src/main/java/com/example/wrk/fragments/ProfileFragment.java (hunk 2 above)
Comment (JCGarzaa) [reply app/src/main/java/com/example/wrk/fragments/ProfileFragment.java:236]: Noted.

**Pull request merged**